### ISSUE TITLE
Dragging mail to Current Folder tab does not open subfolders popup

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/QuickFolders#412]
When dragging an email to the Current folder tab in the center of screen, the menu showing the subfolders is not opened. Expected behavior: the tab should behave like a normal QuickFolders tab and suggest all subfolders in a popup menu. Using the debug switch 
`extensions.quickfolders.debug.dnd = true`
we can see the following output:

```
QuickFolders {DND} 9:47:55.305  [5163 ms]   
 buttonDragObserver.dragEnter - sourceNode = tr
  ALT = false  CTRL = false  SHIFT = false quickfolders-util.js:1299:13
QuickFolders {DND} 9:47:55.371  [66 ms]   
 dragSession.isDataFlavorSupported(text/x-moz-message): false
Drag types from event.dataTransfer:
text/plain
text/x-moz-message
text/x-moz-url
application/x-moz-file-promise-url
application/x-moz-file-promise
application/x-moz-file-promise-dest-filename quickfolders-util.js:1299:13
QuickFolders {DND} 9:47:55.376  [5 ms]   
 showPopup with id moveTo_mailbox://axelg@pop.vodafone.ie/Development/Mozilla/Addons/Quickfolders quickfolders-util.js:1299:13
QuickFolders 9:47:55.377  [1 ms]   
 Document did not return the popup: moveTo_mailbox://axelg@pop.vodafone.ie/Development/Mozilla/Addons/Quickfolders quickfolders-util.js:1304:13
QF(dnd) - no additional popup data null moveTo_mailbox://axelg@pop.vodafone.ie/Development/Mozilla/Addons/Quickfolders quickfolders.js:1676:25

```

**Comment by RealRaven2000** [comment]
When dragging to a standard tab (pointing to the same folder) which successfully creates the popup as expected we get the following output in error console:

```
QuickFolders {DND} 9:50:17.75  [91697 ms]   
 buttonDragObserver.dragEnter - sourceNode = tr
  ALT = false  CTRL = false  SHIFT = false quickfolders-util.js:1299:13
QuickFolders {DND} 9:50:17.97  [22 ms]   
 dragSession.isDataFlavorSupported(text/x-moz-message): false
Drag types from event.dataTransfer:
text/plain
text/x-moz-message
text/x-moz-url
application/x-moz-file-promise-url
application/x-moz-file-promise
application/x-moz-file-promise-dest-filename quickfolders-util.js:1299:13
QuickFolders {DND} 9:50:17.109  [12 ms]   
 showPopup with id moveTo_mailbox://axelg@pop.vodafone.ie/Development/Mozilla/Addons/Quickfolders quickfolders-util.js:1299:13
QuickFolders {DND} 9:50:17.137  [28 ms]   
 set global popup id = moveTo_mailbox://axelg@pop.vodafone.ie/Development/Mozilla/Addons/Quickfolders quickfolders-util.js:1299:13
QuickFolders {DND} 9:50:17.939  [802 ms]   
 buttonDragObserver.dragExit
sourceNode=[object HTMLTableRowElement]event.target=[object XULElement] quickfolders-util.js:1299:13
QuickFolders {DND} 9:50:17.939  [0 ms]   
 buttonDragObserver.dragExit - sourceNode = tr quickfolders-util.js:1299:13
```

**Comment by RealRaven2000** [comment]
Fixed in 6.3 - Published 30/10/2023
